chaosgcp/iam/controls/readme.md

```diff
@@ -0,0 +1,27 @@
+## The `chaosgcp.iam.controls.policy` control to Manage temporary, time-bound IAM roles for the specified project and members.
```

Comment:
We don't usually put readme in subpackage. I'd rather we keep at the top-level readme level.

Reply:
I think that's fine then :)